### DISCUSSION TITLE
SOLVED - Raspberry 4 - Docker Install. Where to put Config.json?

### DISCUSSION THREAD

**Discussion by polykoma** [Zero6992/chatGPT-discord-bot#128] [Q&A]
Dear everyone,
after running the bot normally I decided to go ahead to install it in docker.
I got an error when trying to do docker compose as in the guide.

So I went ahead just doing a docker build from the folder on desktop
docker image build .

Config.json file was filled and in the folder.

When running the docker container it fails with an config.json file missing error.

I went ahead. Created a volume. Mounted it to container and moved settings into there.
Didnt worked.

TL:DR
HOW TO ADD config.json properly into the container of the bot after doing a "docker image build ." on a Raspberry?

Thank you and BR
